ci-operator/step-registry/openshift/e2e/azure/ovn/virt/openshift-e2e-azure-ovn-virt-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-e2e-azure-ovn-virt
+  steps:
+    env:
+      OPENSHIFT_INSTALL_EXPERIMENTAL_DUAL_STACK: "true"
+      TEST_ARGS: --run \[sig-network\]\[OCPFeatureGate:PersistentIPsForVirtualization\]\[Feature:Layer2LiveMigration\]
+      TEST_SUITE: openshift/conformance
```

Comment:
let's add a new suite, to avoid having to skip when openshift virt is not deployed (in other lanes).